src/ext/command.rs

```diff
@@ -56,6 +56,9 @@ pub struct Command {
     /// `None` means they're cleared from the environment.
     envs: Vec<(String, Option<Value>)>,
 
+    /// Whether to clear the env of the child.
+    env_clear: bool,
```

Comment:
Do we still want the changes to this file? It looks like we're just using `.env_remove("PATH")` in the end, and the changes here aren't needed